addons/binding/org.openhab.binding.lutron/ESH-INF/thing/keypads.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you put the keypad in a separate file, while the others are all in thing-types.xml?